### PR TITLE
Metal bridge over air tiles

### PR DESCRIPTION
**Pull request by RafeHaab**
Construction that allows to build metal grates over empty air tiles.

#### Summary
SUMMARY: Feature "Build metal bridges over open-air tiles"

#### Purpose of change

By request, I'm adding a way to create metallic bridges that require the tile to be empty air.

#### Describe the solution
Add a construction project that does it.

#### Describe alternatives you've considered

Wooden bridges, but someone suggested metal and I liked it better.

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
I don't think these should be added until there's a way to have proper supports.

File: data/json/construction.json [hunk 1]
```diff
@@ -882,6 +882,19 @@
     "components": [ [ [ "pebble", 100 ] ] ],
     "post_terrain": "t_railroad_rubble"
   },
+  {
+    "type": "construction",
+    "description": "Build grate bridge over air",
+    "//": "Build metal grate on air.",
```
Comment (BevapDin) [inline data/json/construction.json]:
```suggestion
```
This is just a comment entry and not necessary. If it does not add any useful information, skip it.

**Comment by RafeHaab** [comment]
> I don't think these should be added until there's a way to have proper supports.

You need some kind of support to start building on the second level, actually. Else how do you even get there?

**Comment by FuelType-Memes** [comment]
I think @anothersimulacrum means adding `"pre_special": "check_support",` otherwise you can build infinitely long metal bridges

**Comment by RafeHaab** [comment]
Ok, what's failing now?

**Comment by FuelType-Memes** [comment]
It says right there in the details. For example JSON style check gives this error right at the bottom:

> data/json/construction.json needs to be linted
> Please read doc/JSON_STYLE.md

File: data/json/construction.json [hunk 2]
```diff
@@ -882,6 +882,20 @@
...
   },
+  {
+    "type": "construction",
+    "description": "Build grate bridge over air",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 5 ] ],
+    "time": "60 m",
+    "tools": [ [ [ "oxy_torch", 4 ], [ "welder", 20 ], [ "welder_crude", 30 ], [ "toolset", 30 ] ] ],
+	  "qualities": [ [ { "id": "SAW_M", "level": 1 }, { "id": "GLARE", "level": 2 } ] ],
+    "components": [ [ [ "pipe", 12, "sheet_metal", 1 ] ] ],
```
Comment (kevingranade) [inline data/json/construction.json]:
```suggestion
    "components": [ [ [ "pipe", 12 ] ], [ [ "sheet_metal", 1 ] ] ],
```

**Pull request closed without merging**